core/src/main/java/uk/ac/bbsrc/tgac/miso/core/service/printing/Driver.java

```diff
@@ -121,8 +121,8 @@ public String encode(Barcodable barcodable, int copies) {
 
       offset.addAndGet(multiline(9, 4, barcodable.getLabelText(), writer));
       font.set(3);
-      offset.addAndGet(multiline(9, 2, StringEscapeUtils.unescapeHtml(barcodable.getBarcodeSizeInfo()), writer));
-      multiline(9, 7 - offset.get(), StringEscapeUtils.unescapeHtml(barcodable.getBarcodeExtraInfo()), writer);
+      offset.addAndGet(multiline(9, 2, unescapeHtml(barcodable.getBarcodeSizeInfo()), writer));
+      multiline(9, 7 - offset.get(), unescapeHtml(barcodable.getBarcodeExtraInfo()), writer);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') "NullPointerException" will be thrown when invoking method "multiline()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)